core/db/migrate/20160314162016_add_valid_from_to_spree_prices.rb

```diff
@@ -0,0 +1,5 @@
+class AddValidFromToSpreePrices < ActiveRecord::Migration
+  def change
+    add_column :spree_prices, :valid_from, :datetime
```

Comment:
Shouldn't we add `default: "NOW()"` here, so we can remove the `after_initialize` from price?